fth/file.fth

```diff
@@ -0,0 +1,83 @@
+\ READ-LINE and WRITE-LINE
+\
+\ This file is in the public domain.
+\
+
+private{
+
+10 constant \n
+13 constant \r
+
+\ Unread one char from file FILEID.
+: UNREAD ( fileid -- ior )
+    { f }
```

Comment:
I know, but I still have to teach my editor to indent that correctly. So far it only recognizes the standard syntax {:  :}.